src/store/profiles/actions.js

```diff
@@ -3,7 +3,7 @@
 export const signUp = async function ({ commit }, profileData) {
   const actions = []
 
-  await getProfile({ commit })
+  //await getProfile({ commit })
```

Comment:
This should be removed instead of commented to avoid accumulating non-necessary code.

```suggestion
```